perun-core/src/main/resources/schema.sql

```diff
@@ -857,7 +857,7 @@ create table tasks (
 	status varchar(16) not null,
 	start_time timestamp,
 	end_time timestamp,
-	engine_id integer not null,
+	engine_id integer,
```

Comment:
Please update schema in all required files in ``perun-db`` module and also in all changelogs in ``perun-core``. Also please bump DB version.
There is ``engine_id`` column in ``tasks_results`` table too! I think we should keep it consistent.